sdcm/utils/adaptive_timeouts/__init__.py

```diff
@@ -55,6 +55,7 @@ class Operations(Enum):
     DECOMMISSION = ("decommission", _get_decommission_timeout, ())
     NEW_NODE = ("new_node", _get_soft_timeout, ("timeout",))
     CREATE_INDEX = ("create_index", _get_soft_timeout, ("timeout",))
+    START_SCYLLA = ("create_index", _get_soft_timeout, ("timeout",))
```

Comment:
Typo in the first argument

Reply:
right, not total clear why we aren't using the enum name for this purpose...